staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper_test.go

```diff
@@ -314,6 +379,59 @@ func TestRoundTripAndNewConnection(t *testing.T) {
 					defer proxy.Close()
 				}
 
```

Comment:
move `interceptor := &Interceptor{}` here to keep it local to each test so we don't get cross-test pollution, and pass it into `socks5ProxyServerFunc`

Reply:
moved